FreeRTOS/FreeRTOS
# Add SMP schedule affinity on target test

Add SMP schedule affinity on target test

Description
-----------
Adding the on target test to verify that
>The scheduler shall not schedule a task that is pinned to a specific core on any other core.

Test Steps
-----------
Test result
```
./FreeRTOS/FreeRTOS/Test/Target/tests/smp/schedule_affinity/schedule_affinity.c:227:Test_ScheduleAffinity:PASS

-----------------------
1 Tests 0 Failures 0 Ignored
OK
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
